### PR TITLE
Bz1337217 -- describe how to modify scheduler policy

### PR DESCRIPTION
**Pull request by tnguyen-rh**
Fixes 1337217

@rjhowe
This patch follows the [doc plan](https://bugzilla.redhat.com/show_bug.cgi?id=1337217#c1), which itself follows the BZ [Description](https://bugzilla.redhat.com/show_bug.cgi?id=1337217#c0), which you wrote.  WDYT?

@bfallonf @tpoitras @ahardin-rh @adellape 
Peer review, please?

### CODE REVIEW

File: admin_guide/scheduler.adoc [hunk 1]
```diff
@@ -215,14 +215,34 @@ Each of the priority function has a weight of '1' applied to it:
...
+
+The scheduler policy is defined in a file on the master,
+named *_/etc/origin/master/scheduler.json_* by default,
+unless overridden by the `kubernetesMasterConfig.schedulerConfigFile`
+field in the
+xref:../install_config/master_node_configuration.adoc#master-configuration-files[master configuration file].
+
+To modify the scheduler policy:
+
+. Edit the scheduler configuration file to set predicates and priority functions.
```
Comment (bfallonf) [inline admin_guide/scheduler.adoc]: Should there be an example here? What predicates and priority functions are they setting?

**Comment by bfallonf** [comment]
@tnguyen-rh One question that might need a bit more thinking, but the rest looks good :100:

**Comment by tnguyen-rh** [comment]
@bfallonf Thanks for the review.
I don't think an example is necessary, however i've installed a change that links to the immediately preceding section for the link text "predicates and priority functions".

**Comment by tnguyen-rh** [comment]
@sferich888 Since bug 1268035 (which you filed) is a duplicate of this one, PTAL.

File: admin_guide/scheduler.adoc [hunk 2]
```diff
@@ -215,14 +216,35 @@ Each of the priority function has a weight of '1' applied to it:
...
+
+. Edit the scheduler configuration file to set the desired
+xref:#default-scheduler-policy[predicates and priority functions].
+
+. Restart the {product-title}
+xref:../install_config/master_node_configuration.adoc#launching-servers-using-configuration-files[master services]
+for the changes to take effect.
+
 == Use Cases
 One of the important use cases for scheduling within OpenShift is to support
```
Comment (ahardin-rh) [inline admin_guide/scheduler.adoc]: s/OpenShift/{product-title}

File: admin_guide/scheduler.adoc [hunk 3]
```diff
@@ -215,14 +216,35 @@ Each of the priority function has a weight of '1' applied to it:
...
+
+To modify the scheduler policy:
+
+. Edit the scheduler configuration file to set the desired
+xref:#default-scheduler-policy[predicates and priority functions].
+
+. Restart the {product-title}
+xref:../install_config/master_node_configuration.adoc#launching-servers-using-configuration-files[master services]
+for the changes to take effect.
+
```
Comment (ahardin-rh) [inline admin_guide/scheduler.adoc]: Maybe add a unique anchor here? For example, [[scheduler-use-cases]].

**Comment by ahardin-rh** [comment]
@tnguyen-rh Looks good! Just two minor nits from me.

**Comment by tnguyen-rh** [comment]
@ahardin-rh Thanks for pointing those out.  Although they are valid and useful suggestions, they're not technically part of this BZ/PR, so i've created https://github.com/openshift/openshift-docs/issues/2158 (which @vikram-redhat will probably turn into a BZ if we ask him nicely).

**Comment by sferich888** [comment]
Are we not adding better example of how ow the Scheduler can be configured?

This update simply seems to say, when making changes restart to load what you have changed.

**Comment by tnguyen-rh** [comment]
@sferich888
There are quite a few examples in the section "Sample Policy Configurations" (in the same file).  I've added a link to that section from the new text.  However, now i have a doubt: The samples are all in YAML format, rather than JSON.  Is that OK?

**Comment by sferich888** [comment]
We should have consistency. But it's not not a requirement.

**Comment by tnguyen-rh** [comment]
@rjhowe @sferich888 @bfallonf @ahardin-rh
Thanks for the reviews.

I've opened #2158 and #2211 for spin-off work.

[rev_history]
|xref:../admin_guide/scheduler.adoc#[Scheduler]
|Added a xref:../admin_guide/scheduler.adoc#modifying-scheduler-policy[Modifying Scheduler Policy] section.
%

**Pull request merged**